python/ambassador/ir/irhost.py

```diff
@@ -383,7 +383,8 @@ def matches_httpgroup(self, group: 'IRHTTPMappingGroup') -> bool:
         else:
             # It is NOT A TYPO that we use group.get("host") here -- whether the Mapping supplies
             # "hostname" or "host", the Mapping code normalizes to "host" internally.
-            group_glob = group.get('host') or None  # NOT A TYPO: see above.
+            redirect_glob = group.get('host_redirect', {}).get('host') if group.get('host_redirect', {}) else None
+            group_glob = group.get('host') or redirect_glob or None  # NOT A TYPO: see above.
```

Comment:
If `redirect_glob` is falsey, doesn't that mean it's `None`? Isn't the `or None` superfluous?

Reply:
This is true, I can definitely change that